.circleci/config.yml

```diff
@@ -3,15 +3,14 @@ version: 2
 references:
   container: &container
     docker:
-      - image: circleci/node:8.10-browsers
+      - image: circleci/node:10.15.1-browsers
         environment:
           CHROME_PATH: /usr/bin/google-chrome
 
   restore_deps: &restore_deps
     restore_cache:
       keys:
-        - v1-uswds-dependencies-{{ checksum "package-lock.json" }}
-        - v1-uswds-dependencies-
+        - v2-uswds-dependencies-{{ checksum "package-lock.json" }}
```

Comment:
Working now after forcing the cache to update by changing the key. Removed `v1-uswds-dependencies-` bc it was using that cache even if `v2-uswds-dependencies-{{ checksum "package-lock.json" }}` was different, but would like someone to check this.